scripts/benchmark_stats.py

```diff
@@ -30,15 +32,25 @@ def __str__(self):
             f"{percentile}th: {self.percentiles[percentile]}ms"
             for percentile in self.PERCENTILES_TO_REPORT
         )
+        formatted_metrics = {
```

Comment:
Could the formatted metrics and percentiles maybe move to the init after the file is processed? Might make this a bit tidier 🤔

Reply:
Updated 👍